plugins/modules/zos_operator.py

```diff
@@ -35,10 +35,28 @@
     default: false
   debug:
     description:
-      - Return debugging information.
+      - Return rexx debugging information.
```

Comment:
My thoughts are that debugging should just be so, users don't need to know its REXX driving the operator commands.